config.yml

```diff
@@ -1,51 +1,23 @@
 experiment_name: my_example
```

Comment:
I think we talked about it already: you have to keep `config.yml` as is (5 repeats, MNIST and CIFAR10, etc.). If you want, you can add Titanic to it.

Reply:
Yes, I will revert the `config.yml` file.
Since the performance are not great with this model I rather keep it as an available exemple rather than a technical demo.